tools/test-backend

```diff
@@ -123,7 +123,7 @@ def block_internet():
                         "https://zulip.readthedocs.io/en/latest/testing.html#internet-access-inside-test-suits")
 
     httplib2.Http.request = internet_guard
-    requests.request = internet_guard
+    requests.request = internet_guard  # type: ignore # FIXME file bug in mypy repo!
```

Comment:
I think this actually should be a `# type: ignore # monkey-patching`, since we are blatantly replacing  afunction with a different type?.